definitions/modules/fuel.yaml

```diff
@@ -11,18 +11,6 @@ shared:
     var_name: A:GENERAL ENG FUEL PUMP SWITCH:2
     var_units: Bool
     event_name: TOGGLE_ELECT_FUEL_PUMP2
-  -
-    type: NumSet
-    var_name: A:GENERAL ENG FUEL PUMP SWITCH EX1:1
-    var_units: Enum
-    var_type: i32
-    event_name: ELECT_FUEL_PUMP1_SET
-  -
-    type: NumSet
-    var_name: A:GENERAL ENG FUEL PUMP SWITCH EX1:2
-    var_units: Enum
-    var_type: i32
-    event_name: ELECT_FUEL_PUMP2_SET
```

Comment:
This might cause issues with other aircraft because I believe some use more than 0 and 1, hence the Enum. If non-EX1 works on aircraft that use EX1, then just change non-EX1's to NumSet.

Reply:
I had to remove one of these as they would conflict, so we either need to separate them into each individual profile or verify that the EX works with the non-ex.